docs/source/dlv2_tutorial.rst

```diff
@@ -50,3 +50,21 @@ Distributed
         for d in dl:
             model(d)
     dl.shutdown()
+
+Multiprocessing + Distributed
```

Comment:
Non-blocking but it will be nice to add this to our Colab example as well!

Reply:
Sounds reasonable to me.